src/pages/Insider/Stories/Stories.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Button, Grid } from "grommet";
+import { 
+    CelebrateTeam,
+    CodeWars, 
+    EarningsQ2, 
+    HPEDiscover, 
+    HPENvidia,
+    ResourceGroupEvents
+} from "./stories/index";
+import { LinkDown } from "grommet-icons";
+
+const areas = [
+    ['story-1', 'story-1', 'story-2', 'story-2'],
+    ['story-3', 'story-4', 'story-5', 'story-6']
+];
+
+const columns = ['flex', 'flex', 'flex', 'flex'];
```

Comment:
As a refinement here (to help recommend best practices) we could include responsive grid behavior.

Not required at this moment.

Reply:
Can we get this in and then build atop?